packages/discovery-react-components/src/components/CIDocument/components/CIDocumentContent/CIDocumentContent.tsx

```diff
@@ -182,4 +175,13 @@ function scrollToActiveItem(
   );
 }
 
+function getHighlightLocationId(highlightWithMeta: HighlightWithMeta): string {
+  return getLocationId({
+    location: {
+      begin: highlightWithMeta.begin,
+      end: highlightWithMeta.end
+    }
+  });
+}
```

Comment:
Could replace this by condensing `begin` and `end` inside a `location` field. Lots of ways to make this cleaner, but it was such a little thing I haven't stopped to choose and implement one of them.